styleguide/components/notice-banner.js

```diff
@@ -0,0 +1,17 @@
+import { text } from '@storybook/addon-knobs'; // eslint-disable-line
+import wrapper from './component-wrapper';
+
+const noticeBanner = () => {
+    const content = text(
+        'Banner content',
+        'If you’re a Thomas Cook customer and you’re stuck abroad or want to get your money back, get help from the Civil Aviation Authority.'
+    );
+
+    const component = `<div class="cads-notice-banner">
+    <h4>Notice</h4>
```

Comment:
I think this needs to be a span and not a heading. Does it make sense to have `<span class="h4">`

Reply:
Depends how it would be read out, if span is more appropriate we can do the span with h4 style.

Reply:
Also, the large version is just for the large breakpoint or md/lg?